### DISCUSSION TITLE
Why does it say to not change the macro.cfg?

### DISCUSSION THREAD

**Discussion by oconnellrv** [dz0ny/klipper-prusa-mk3s#22] [Q&A] [answered]
```
# WARNING. DO NOT EDIT THIS FILE.
# To override settings from this file, you can copy and paste the relevant
# sections into your printer.cfg and change it there.
```

Just curious, I'd like to add some kind of delay to the m600 macro.  I can enter it into the printer.cfg file but why not the macro.cfg? 

I just had to scrap an 8 hour print because the m600 macro paused the print, and immediately turned off the bed and wanted me to home the xyz axis.  I thought I would trick it by swapping sheets, homing and swapping back but then I got a thermal shutdown.

**Comment by dz0ny** [comment, marked as answer]
macro.cfg is managed by git, so every time we would make a change in the repo your change would get lost. You can copy the macro.cfg out of git managed folder, change the path in printer.cfg to your local one and then you will be safe during updates

**Reply by oconnellrv** [reply]
Make sense, thank you.  Turns out I just needed to add an idle timeout to printer.cfg
